src/bidiMapper/domains/context/cdpTarget.ts

```diff
@@ -115,6 +124,9 @@ export class CdpTarget {
     });
 
     await this.#cdpClient.sendCommand('Runtime.runIfWaitingForDebugger');
+
+    await this.loadTopLevelPreloadScripts();
```

Comment:
It's not always top-level. It can be preload scripts for it's parent context.

Reply:
Please add e2e for such a case

Reply:
You can move it to another PR though, but please add a failing e2e anyways in this PR.

Reply:
As discussed during live code review, we already have such a test in `test_preload_script.py`, which is currently skipped.